geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/Utils.scala

```diff
@@ -61,8 +61,8 @@ object Utils {
   object Formats {
     val CSV     = "csv"
     val TSV     = "tsv"
-    val TIFF    = "tiff"
-    val DTED    = "dted"
+    val TIFF    = "GTiff"
```

Comment:
GTiff is the abbreviation used in GDAL for geotiff, not that we need to conform to that. Either way is fine.